.travis.settings.xml

```diff
@@ -0,0 +1,10 @@
+<settings>
+  <mirrors>
+    <mirror>
+      <id>google-maven-central</id>
+      <name>Google Maven Central</name>
+      <url>https://maven-central.storage.googleapis.com/repos/central/data/</url>
```

Comment:
We use this mirror no Kororo because we're on GCE with Kokoro. The container-based Trusty is [currently on EC2 or Packet.net](https://docs.travis-ci.com/user/reference/overview/), so it isn't as good of an option. So I'd suggest we [copy the Travis configuration](https://github.com/grpc/grpc-java/pull/4813#issuecomment-418514808) but only with central + sonatype repos. If we want, we can have the second or third be the Google mirror.  But I'd want to keep the default/first mirror still repo.maven.apache.org.